src/Analyzers/Core/Analyzers/AbstractBuiltInCodeStyleDiagnosticAnalyzer.cs

```diff
@@ -114,15 +120,17 @@ internal abstract class AbstractBuiltInCodeStyleDiagnosticAnalyzer : AbstractCod
         /// Message for the diagnostic descriptor.
         /// Null if the message is identical to the title.
         /// </param>
+        /// <param name="isUnnecessary"></param>
```

Comment:
Does the unnecessary custom tag have any semantic meaning associated with it now that we check the diagnostic properties to determine if it is an unnecessary diagnostic not the custom tags?

Reply:
We use both. The diagnostic tag determines whether the primary location is faded, and the properties cover additional locations.

Reply:
Ah got you, thanks!